coverage.py

```diff
@@ -126,7 +126,14 @@ def __init__(self, collection, mirror=None, http_get=None, viaf=None,
 
```

Comment:
The changes to this file are already in master; I don't know why they're showing up in this diff.

Reply:
Merging master into the branch would probably fix it.